custom/icds_reports/management/commands/sql_scripts/fix_past_data_part_1_1.sql

```diff
@@ -0,0 +1,42 @@
+UPDATE child_health_monthly
```

Comment:
how is fix_past_data_part_1_1.sql different from fix_past_data_part_1.sql

Reply:
it was done to make looping thing easy in the script